extension/source/Controllers/Network/web3_clientVersion.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const web3_clientVersion = 'Quill/v0.1.0';
+
+export default web3_clientVersion;
```

Comment:
~Lint disable seems like overkill vs. using a more standard name.~ I see the usage point now.
Also, you could pull the version segment from `package.json` so we don't have to track in multiple places.

```suggestion
import { version } from '../../../package.json';
// Match createScaffoldMiddleware payload name
// eslint-disable-next-line @typescript-eslint/naming-convention
const web3_clientVersion = `Quill/v${version}`;

export default web3_clientVersion;
```

Reply:
done